src/app/models/resource-info.model.ts

```diff
@@ -19,4 +19,5 @@
 export interface ResourceInfo {
   memory: string;
   vcore: string;
+  pods?: string;
```

Comment:
I assume that this is an allocation on a node. I don't think we should call this a pod. A pod is closely linked to K8s and is not transferable to other managers.
This does however open up a new point for me: the scheduler supports any resource name. It does not have any resource names coded beside the two mentioned here for test cases. We need to think about how that translates to the web UI.

Reply:
In nodes response, there are a bunch of resources(pods, aws ebs etc) in addition to memory and vcore and its a long list. We can display the resources as what UI gets from scheduler, not sure how would that make sense in UI.

Reply:
hi @akhilpb001 , is it too bad to have them all displayed on UI? I think we need to do some cleanup at the scheduler side to focus on a whitelist of resource types, instead of tracking them all. If it is not looking too bad on UI, we can get this merged first and then work on the cleanup separately.

Reply:
Agree with that solution.
From a UI perspective it should just display what is provided by the REST interface. Filtering should be done on the provider side not the display side.

Reply:
@wilfred-s  @yangwwei  I have tried to display the entire resources in the UI. Its too long and not looking good in the UI. So for now I have kept to memory and vcore. We will fix this another jira.

Reply:
logged [YUNIKORN-138](https://issues.apache.org/jira/browse/YUNIKORN-138) for this